hyp3_gamma/rtc/rtc_sentinel.py

```diff
@@ -35,11 +37,26 @@
 from hyp3_gamma.rtc.coregistration import CoregistrationError, check_coregistration
 from hyp3_gamma.util import set_pixel_as_point, unzip_granule
 
+
 log = logging.getLogger()
 gdal.UseExceptions()
 ogr.UseExceptions()
 
 
+def createPowerDB(fi, nodata=None):
+    f = gdal.Open(fi)
+    in_nodata = f.GetRasterBand(1).GetNoDataValue()
+    _, _, trans, proj, data = saa.read_gdal_file(f)
+    data = np.ma.masked_less_equal(np.ma.masked_values(data, in_nodata), 0.)
+    powerdb = 10*np.ma.log10(data)
+    if not nodata:
+        nodata = np.finfo(data.dtype).min.astype(float)
+    outfile = fi.replace('.tif', '-db.tif')
```

Comment:
I'd consider taking `outfile` as a function parameter.  That way we can skip creating an intermediate file when calling this function around line 382:

```
        elif scale == 'decibel':
            decibel_tif = createPowerDB(power_tif, output_tif)
```